R/auth.R

```diff
@@ -220,6 +221,56 @@ showInvited <- function(appDir=getwd(), appName=NULL, account = NULL,
   return(df)
 }
 
+#' Resend invitation for invited users of an application
+#'
+#' @param invite The invitation to resend. Can be id or email address.
+#' @param regenereate Regenerate the invite code. Can be helpful is the
+#' invitation has expired.
+#' @param appDir Directory containing application. Defaults to
+#'   current working directory.
+#' @param appName Name of application.
+#' @param account Account name. If a single account is registered on the
+#'   system then this parameter can be omitted.
+#' @param server Server name. Required only if you use the same account name on
+#'   multiple servers.
+#' @seealso [showInvited()]
+#' @note This function works only for ShinyApps servers.
+#' @export
+resendInvitation <- function(invite, regenerate=FALSE,
+                             appDir=getwd(), appName=NULL,
+                             account = NULL, server=NULL) {
+
+  # resolve account
+  accountDetails <- accountInfo(resolveAccount(account, server), server)
+
+  # get invitations
+  invited <- showInvited(appDir, appName, account, server)
+
+  if (is.numeric(invite)) {
```

Comment:
Is there any point to handling a vector of invitations?

Reply:
Potentially useful, but it would be inconsistent with the other APIs. 🤷‍♂️

Reply:
kk. as-is it is, then! let me know once you're happy with testing and we'll get this merged.

Reply:
Any idea why Travis is complaining? The output looks like CRAN check non-sense to me. Are these transient failures?

Reply:
No clue, sadly, and other recent PRs passed.

Reply:
When in doubt, regenereate! 🎉

Reply:
I think the travis output is being truncated, but I found this running it locally.